pkg/morph/client/multi.go

```diff
@@ -63,6 +61,16 @@ func (c *Client) switchRPC() bool {
 			continue
 		}
 
+		c.client = cli
+		c.rpcActor = act
+		c.gasToken = gas
+
+		if !c.switchIsActive.Load() &&
+			c.endpoints.list[c.endpoints.curr].Priority != c.endpoints.list[0].Priority {
+			c.switchIsActive.Store(true)
+			go c.switchToMostPrioritized()
```

Comment:
Do we create a goroutine each time `switchRPC` is called? I think this should be a background worker.

Reply:
> Do we create a goroutine each time switchRPC is called?

yes if it has not been created yet: my idea was that a node with a non-prioritized connection is smth really rare, so i do not want to have a goroutine if i know that i do not need it

sounds strange but that was a 1-month-younger me. what do you think?